themes-default/slim/src/store/modules/recommended.js

```diff
@@ -0,0 +1,62 @@
+import { ADD_RECOMMENDED_SHOW } from '../mutation-types';
+
+const state = {
+    recommended: []
+};
+
+const mutations = {
+    [ADD_RECOMMENDED_SHOW](state, show) {
```

Comment:
@OmgImAlexis I could really use some help with the store modules. I got it working somehow and somewhat, through some copy/pasting.